### ISSUE TITLE
TB 44.0a2 not working with conversation view

### ISSUE THREAD

**Issue by darrenkenny** [protz/thunderbird-conversations#1022]
I recently installed TB 44.0a2 and noticed that I no longer was seeing conversations.

If I disable Conversation View in the menu, I can see e-mails again, but otherwise it just
shows up as blank, and never marks the e-mail as read - so it seems to never load it.

I tried the latest download from your personal wiki and the one before that, but neither made
any difference.

In the error console, when I toggle Conversation View on, I get the following messages:
```
Timestamp: 02/11/2015 11:26:29
Error: ReferenceError: jQuery is not defined
Source File: chrome://conversations/content/js/jquery.tokeninput.js
Line: 639

Timestamp: 02/11/2015 11:26:29
Error: TypeError: redeclaration of let Log
Source File: chrome://conversations/content/stub.completion-ui.js
Line: 20

Timestamp: 02/11/2015 11:26:29
Error: TypeError: redeclaration of let Log
Source File: chrome://conversations/content/stub.xhtml
Line: 39
```

(This is running 2.11pre)

**Comment by msprotz** [comment]
Right. I've been running aurora because Thunderbird itself was broken. Time for me to switch back to Daily.

**Comment by protz** [comment]
There's no trivial fix in sight (tried fixed this last weekend). Don't hold your breath.

**Comment by protz** [comment]
Fixed by @shirosaki in #1033 (and the one before that).
